aspnetcore/mvc/views/tag-helpers/builtin.md

```diff
@@ -11,7 +11,7 @@ Microsoft has included 17 Tag Helpers that can be immediately used in .net Core
 
 **[AnchorTagHelper](builtin/resources/AnchorTagHelper.md)**
 
-[comment]: **[CacheTagHelper](builtin/resources/AnchorTagHelper.md)**
+**[CacheTagHelper](builtin/resources/CacheTagHelper.md)**
```

Comment:
Should the docs and the link text be titled with spaces? It would read better imo *Anchor Tag Helper*, *Cache Tag Helper*, etc.